.env

```diff
@@ -1,6 +1,8 @@
-DB_URL=mongodb://127.0.0.1:27017/e-commerce
 NODE_ENV=development
 PORT=8080
 DB_MAX_POOL_SIZE=100
 DB_MIN_POOL_SIZE=10
 DB_SELECTION_TIMEOUT=30000
+DB_URL=mongodb://mongodb:27017/e-commerce
```

Comment:
Dont replace this with docker uri, instead create a seperate variable like DOCKER_DB_URL, also keep the DB_URL so that we can run it locally as well, same applies to all other variables

Reply:
I've reverted the DB_URL and when application will be brought up using docker image then I've configured in docker compose to override DB_URL to point to the mongo container.

Removed other two mongo variables